ios/NativeSigner/Screens/KeySetsList/KeySetRow.swift

```diff
@@ -66,7 +67,7 @@ struct AddressCardSelector_Previews: PreviewProvider {
                 )
             )
         }
-        .preferredColorScheme(.dark)
+//        .preferredColorScheme(.dark)
```

Comment:
is it intentional?

Reply:
`_Preview` / `_Previews` are just for mocking UI states for live-preview feature of XCode 13 / SwiftUI.
I played around with tests for light mode, that's why it's commented out.

I'll go ahead and comment out all previews and will keep it like this when pushing code to repo.

I've initially uncommented all previews (and new ones) as this way it's easy to make sure they don't go out of date (i.e. if `init` requirements are changing)